rpc/send.go

```diff
@@ -86,6 +86,29 @@ func (s SendError) Error() string {
 // CanRetry implements the Retryable interface.
 func (s SendError) CanRetry() bool { return s.canRetry }
 
+//StoreAddr is used to identify the address of replication
+//One node may have multiple stores, so StoreID is needed
+type StoreAddr struct {
+	Addr    net.Addr
+	StoreID proto.StoreID
+}
+
+func (addr *StoreAddr) String() string {
+	return addr.Addr.String() + fmt.Sprint(addr.StoreID)
```

Comment:
Wondering if this is sufficient or if you need to add a separator that can never occur in `addr.Addr.String()`. For example, what happens if there are 2 nodes with addresses `<ip>:1000` and `<ip>:10001`. If store id 1 is located on the first node then `StoreAddr.String` would return `<ip>:10001` which be confused with the second node. Unlikely a problem, but better to do something like:

```
  return fmt.Sprintf("%s#%d", addr.Addr, addr.StoreID)
```